### PR TITLE
CT-1768 redesign report page

### PR DESCRIPTION
**Pull request by aliuk2012**
- Updates stats home page
- Moves audit csv to the admin dashboard
- moves the business unit map to the bottom of the team pages

### CODE REVIEW

File: db/migrate/20180621094208_add_correspondence_type_columns_to_report_types.rb [hunk 1]
```diff
@@ -4,8 +4,6 @@ class AddCorrespondenceTypeColumnsToReportTypes < ActiveRecord::Migration[5.0]
   def up
     add_column :report_types, :foi, :boolean, default: false
     add_column :report_types, :sar, :boolean, default: false
-    ReportType.reset_column_information
```
Comment (beccapearce) [inline db/migrate/20180621094208_add_correspondence_type_columns_to_report_types.rb]: Bit worried about removing the seeder bit in this migration

**Review by beccapearce — COMMENTED** [review]
All looks good, but want to check about moving the seeder out of the migration, could be fine but want to double check :)

File: db/migrate/20180621094208_add_correspondence_type_columns_to_report_types.rb (hunk 1 above)
Comment (stephenrichards) [reply db/migrate/20180621094208_add_correspondence_type_columns_to_report_types.rb]: Adding a column to a table, and updating the seeds to populate that column cause problems when running rake db:reseed.  The way to do it is to maintain a strict separation of schema migrations and data migrations, so I've moved them all into data migrations.

Reseeding will run all the schema migrations, and then all the data migrations which will ensure everything is populated.  Ditto on production systems, we just run all the outstanding schema migrations first, followed by all the outstanding data migrations.

File: app/models/report_type.rb [hunk 2]
```diff
@@ -2,26 +2,57 @@
...
 
 class ReportType < ApplicationRecord
+
+  VALID_DEFAULT_REPORTING_PERIODS = %w{
+                                        year_to_date
+                                        quarter_to_date
+                                        last_quarter
+                                        last_month
+                                      }
+
```
Comment (misaka) [inline app/models/report_type.rb]: It'd be nice to have this as an `enum`. It appropriately descriptive and would give us scopes and validation for free.

File: app/services/stats/base_appeals_performance_report.rb [hunk 3]
```diff
@@ -15,10 +15,10 @@ class BaseAppealsPerformanceReport < BaseReport
         responsible:                     ''
     }
 
-    def initialize(period_start= Time.now.beginning_of_year, period_end=Time.now)
+    def initialize(period_start = nil, period_end = nil)
```
Comment (misaka) [inline app/services/stats/base_appeals_performance_report.rb]: `period_start` and `period_end` are not used in this method. And is there a reason to keep the commented-out assignments to `@period_start` and `@period_end` below?

**Review by stephenrichards — APPROVED** [review]

File: app/services/stats/base_appeals_performance_report.rb (hunk 3 above)
Comment (misaka) [inline app/services/stats/base_appeals_performance_report.rb]: How about prefixing these with underscore ('_') to denote that they're intentionally unused?

**Review by misaka — APPROVED** [review]

**Pull request merged**